pyproject.toml

```diff
@@ -27,6 +27,7 @@ dependencies = [
     'coverage',
     "deprecated",
     "ipykernel", # Support for Jupyter notebooks
+    "light-curve>=0.7.3,<0.8.0",
```

Comment:
Out of curiosity, what's the pin to <0.8.0 motivated by, is this something that we would need to change to enable more recent versions of light-curve?

Reply:
I'm trying to follow [semantic versioning](https://semver.org) conventions, so potentially I can brake backward compatibility in the 0.(x+1).0 version. Practically I don't brake it in a terrible or unsound way, but I still would like the version to be restricted.